src/main/kotlin/io/sip3/salto/ce/rtcp/RtcpHandler.kt

```diff
@@ -66,13 +68,61 @@ open class RtcpHandler : AbstractVerticle() {
         }
 
         vertx.eventBus().localConsumer<Packet>(RoutesCE.rtcp) { event ->
-            try {
-                val packet = event.body()
-                handleRaw(packet)
-            } catch (e: Exception) {
-                logger.error(e) { "RtcpHandler 'handleRaw()' failed." }
+            val packet = event.body()
+            if (packet.source == "hep3") {
```

Comment:
Let's replace 2 `if` statements with 1 `when`. And switch 2 conditions because we will have more packets of "sip3" type, so it's more performance wise